tests/testthat/test-chevron_tlg-methods.R

```diff
@@ -194,25 +194,27 @@ test_that("script_funs works as expected", {
 
 
 test_that("script_funs generates a valid script", {
-  tmp <- tempfile()
-
-  args_list <- list(
-    arm_var = "ARM"
-  )
+  withr::with_tempfile("tmp", fileext = ".R", {
```

Comment:
the rest of the code is identical right?

Reply:
Yeah, it is just a cleaner way of using a temporary file in the test.